baybe/recommenders/base.py

```diff
@@ -193,16 +173,18 @@ def _recommend_hybrid(
         """
         raise NotImplementedError("Hybrid recommendation is not implemented.")
 
-    def _select_candidates_and_recommend(
+    def _recommend_with_discrete_parts(
         self,
-        subspace_discrete: SubspaceDiscrete,
+        searchspace: SearchSpace,
         batch_size: int,
     ) -> pd.DataFrame:
-        """Get candidates in a discrete search space and generate recommendations.
+        """Obtain recommendations in search spaces with discrete parts.
+
+        Convenience helper which sequentially performs the following tasks: get discrete
```

Comment:
I personally would prefer proper sentences here, but since this is a private function, I'd also be happy with this.

Reply:
the sentence seems proper to me so unless you point out an alternative I will keep it